src/njoy21/input/LEAPR/Card6.hpp

```diff
@@ -0,0 +1,42 @@
+class Card6 {
+  public:
+  #include "njoy21/input/LEAPR/Card6/Nss.hpp"
+  #include "njoy21/input/LEAPR/Card6/B7.hpp"
+  #include "njoy21/input/LEAPR/Card6/Aws.hpp"
+  #include "njoy21/input/LEAPR/Card6/Sps.hpp"
+  #include "njoy21/input/LEAPR/Card6/Mss.hpp"
+  
+  using ControlTuple = std::tuple< Argument< B7  >, Argument< Aws >, 
```

Comment:
I like that you grouped these dependent arguments together, but let's use an explicit struct in this case, e.g.

```cpp
struct ControlTuple {
  Argument< B7 > b7;
  ...
};
```